### DISCUSSION TITLE
Is the simulation setting for self-extinguishing through water curtain correct?

### DISCUSSION THREAD

**Discussion by joker121-move** [firemodels/fds#13214] [General]
Dear experts and teachers, could you please help me look at this fds code? I want to simulate the self-extinguishing of flames through water curtain blocking. Is my code set correctly? Are there any errors or suggestions in terms of self-extinguishing?

&HEAD CHID='extinguish2'/
&TIME T_END=500.0/
&DUMP RENDER_FILE='extinguish2.ge1', DT_RESTART=1.0, DT_SL3D=0.25/
&MISC CHECK_HT=.TRUE., TURBULENCE_MODEL='DYNAMIC SMAGORINSKY'/
&CLIP CLIP_DT_RESTRICTIONS_MAX=0/

&MESH ID='Mesh1', IJK=30,45,21, XB=0.0,6.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh2', IJK=30,45,21, XB=6.0,12.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh3', IJK=30,45,21, XB=12.0,18.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh4', IJK=20,90,42, XB=18.0,20.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh5', IJK=18,90,8, XB=20.0,21.8,0.0,9.0,4.2,5.0/
&MESH ID='Mesh6', IJK=18,90,21, XB=20.0,21.8,0.0,9.0,2.1,4.2/
&MESH ID='Mesh7', IJK=18,90,21, XB=20.0,21.8,0.0,9.0,0.0,2.1/
&MESH ID='Mesh8', IJK=22,90,42, XB=21.8,24.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh9', IJK=30,45,21, XB=24.0,30.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh10', IJK=30,45,21, XB=30.0,36.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh11', IJK=30,45,21, XB=36.0,42.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh12', IJK=30,45,21, XB=42.0,48.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh13', IJK=30,45,21, XB=48.0,54.0,0.0,9.0,0.0,4.2/
&MESH ID='Mesh14', IJK=30,45,21, XB=54.0,60.0,0.0,9.0,0.0,4.2/

&SPEC ID='WATER VAPOR'/

&PART ID='ignitor particle',
      STATIC=.TRUE.,
      SURF_ID='ignitor'/
&PART ID='Water',
      SPEC_ID='WATER VAPOR',
      DIAMETER=300.0,
      MONODISPERSE=.TRUE.,
      AGE=60.0,
      SAMPLING_FACTOR=1/

&REAC ID='PROPYLENE',
      FYI='FM SNL FDS5 Validation',
      FUEL='REAC_FUEL',
      FORMULA='C7H16',
      AUTO_IGNITION_TEMPERATURE=215.0,
      SOOT_YIELD=0.37,
      RADIATIVE_FRACTION=0.35/

&PROP ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray',
      QUANTITY='SPRINKLER LINK TEMPERATURE',
      ACTIVATION_TEMPERATURE=68.33,
      PART_ID='Water',
      K_FACTOR=80.0,
      OPERATING_PRESSURE=3.0,
      PARTICLE_VELOCITY=5.0,
      SPRAY_ANGLE=0.0,45.0/
&DEVC ID='SPRK1', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.5,0.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK2', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.5,1.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK3', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.5,2.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK4', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.5,3.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK5', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.5,4.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK6', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.5,5.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK7', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.5,6.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK8', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.5,7.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK9', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.5,8.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK10', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.9,1.0,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK11', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.9,2.0,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK12', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.9,3.0,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK13', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.9,4.0,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK14', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.9,5.0,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK15', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.9,6.0,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK16', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.9,7.0,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK17', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=20.9,8.0,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK18', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=21.3,0.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK19', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=21.3,1.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK20', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=21.3,3.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK21', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=21.3,2.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK22', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=21.3,4.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK23', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=21.3,5.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK24', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=21.3,6.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK25', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=21.3,7.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='SPRK26', PROP_ID='Generic Commercial Link_Generic Commercial Link Generic Spray_Generic Storage Spray', XYZ=21.3,8.5,4.7, INITIAL_STATE=.TRUE., LATCH=.FALSE./
&DEVC ID='THCP', QUANTITY='THERMOCOUPLE', XYZ=0.0,4.5,4.137/
&DEVC ID='THCP01', QUANTITY='THERMOCOUPLE', XYZ=0.5,4.5,4.137/
&DEVC ID='THCP02', QUANTITY='THERMOCOUPLE', XYZ=1.0,4.5,4.137/
&DEVC ID='THCP03', QUANTITY='THERMOCOUPLE', XYZ=1.5,4.5,4.137/
&DEVC ID='THCP04', QUANTITY='THERMOCOUPLE', XYZ=2.0,4.5,4.137/
&DEVC ID='THCP05', QUANTITY='THERMOCOUPLE', XYZ=2.5,4.5,4.137/
&DEVC ID='THCP06', QUANTITY='THERMOCOUPLE', XYZ=3.0,4.5,4.137/
&DEVC ID='THCP07', QUANTITY='THERMOCOUPLE', XYZ=3.5,4.5,4.137/
&DEVC ID='THCP08', QUANTITY='THERMOCOUPLE', XYZ=4.0,4.5,4.137/
&DEVC ID='THCP09', QUANTITY='THERMOCOUPLE', XYZ=4.5,4.5,4.137/
&DEVC ID='THCP10', QUANTITY='THERMOCOUPLE', XYZ=5.0,4.5,4.137/
&DEVC ID='THCP11', QUANTITY='THERMOCOUPLE', XYZ=5.5,4.5,4.137/
&DEVC ID='THCP12', QUANTITY='THERMOCOUPLE', XYZ=6.0,4.5,4.137/
&DEVC ID='THCP13', QUANTITY='THERMOCOUPLE', XYZ=6.5,4.5,4.137/
&DEVC ID='THCP14', QUANTITY='THERMOCOUPLE', XYZ=7.0,4.5,4.137/
&DEVC ID='THCP15', QUANTITY='THERMOCOUPLE', XYZ=7.5,4.5,4.137/
&DEVC ID='THCP16', QUANTITY='THERMOCOUPLE', XYZ=8.0,4.5,4.137/
&DEVC ID='THCP17', QUANTITY='THERMOCOUPLE', XYZ=8.5,4.5,4.137/
&DEVC ID='THCP18', QUANTITY='THERMOCOUPLE', XYZ=9.0,4.5,4.137/
&DEVC ID='THCP19', QUANTITY='THERMOCOUPLE', XYZ=9.5,4.5,4.137/
&DEVC ID='THCP20', QUANTITY='THERMOCOUPLE', XYZ=10.0,4.5,4.137/
&DEVC ID='THCP21', QUANTITY='THERMOCOUPLE', XYZ=10.5,4.5,4.137/
&DEVC ID='THCP22', QUANTITY='THERMOCOUPLE', XYZ=11.0,4.5,4.137/
&DEVC ID='THCP23', QUANTITY='THERMOCOUPLE', XYZ=11.5,4.5,4.137/
&DEVC ID='THCP24', QUANTITY='THERMOCOUPLE', XYZ=12.0,4.5,4.137/
&DEVC ID='THCP25', QUANTITY='THERMOCOUPLE', XYZ=12.5,4.5,4.137/
&DEVC ID='THCP26', QUANTITY='THERMOCOUPLE', XYZ=13.0,4.5,4.137/
&DEVC ID='THCP27', QUANTITY='THERMOCOUPLE', XYZ=13.5,4.5,4.137/
&DEVC ID='THCP28', QUANTITY='THERMOCOUPLE', XYZ=14.0,4.5,4.137/
&DEVC ID='THCP29', QUANTITY='THERMOCOUPLE', XYZ=14.5,4.5,4.137/
&DEVC ID='THCP30', QUANTITY='THERMOCOUPLE', XYZ=15.0,4.5,4.137/
&DEVC ID='THCP31', QUANTITY='THERMOCOUPLE', XYZ=15.5,4.5,4.137/
&DEVC ID='THCP32', QUANTITY='THERMOCOUPLE', XYZ=16.0,4.5,4.137/
&DEVC ID='THCP33', QUANTITY='THERMOCOUPLE', XYZ=16.5,4.5,4.137/
&DEVC ID='THCP34', QUANTITY='THERMOCOUPLE', XYZ=17.0,4.5,4.137/
&DEVC ID='THCP35', QUANTITY='THERMOCOUPLE', XYZ=17.5,4.5,4.137/
&DEVC ID='THCP36', QUANTITY='THERMOCOUPLE', XYZ=18.0,4.5,4.137/
&DEVC ID='THCP37', QUANTITY='THERMOCOUPLE', XYZ=18.5,4.5,4.137/
&DEVC ID='THCP38', QUANTITY='THERMOCOUPLE', XYZ=19.0,4.5,4.137/
&DEVC ID='THCP39', QUANTITY='THERMOCOUPLE', XYZ=19.5,4.5,4.137/
&DEVC ID='THCP40', QUANTITY='THERMOCOUPLE', XYZ=20.0,4.5,4.137/
&DEVC ID='THCP41', QUANTITY='THERMOCOUPLE', XYZ=20.5,4.5,4.137/
&DEVC ID='THCP42', QUANTITY='THERMOCOUPLE', XYZ=21.0,4.5,4.137/
&DEVC ID='THCP43', QUANTITY='THERMOCOUPLE', XYZ=21.5,4.5,4.137/
&DEVC ID='THCP44', QUANTITY='THERMOCOUPLE', XYZ=22.0,4.5,4.137/
&DEVC ID='THCP45', QUANTITY='THERMOCOUPLE', XYZ=22.5,4.5,4.137/
&DEVC ID='THCP46', QUANTITY='THERMOCOUPLE', XYZ=23.0,4.5,4.137/
&DEVC ID='THCP47', QUANTITY='THERMOCOUPLE', XYZ=23.5,4.5,4.137/
&DEVC ID='THCP48', QUANTITY='THERMOCOUPLE', XYZ=24.0,4.5,4.137/
&DEVC ID='THCP49', QUANTITY='THERMOCOUPLE', XYZ=24.5,4.5,4.137/
&DEVC ID='THCP50', QUANTITY='THERMOCOUPLE', XYZ=25.0,4.5,4.137/
&DEVC ID='THCP51', QUANTITY='THERMOCOUPLE', XYZ=25.5,4.5,4.137/
&DEVC ID='THCP52', QUANTITY='THERMOCOUPLE', XYZ=26.0,4.5,4.137/
&DEVC ID='THCP53', QUANTITY='THERMOCOUPLE', XYZ=26.5,4.5,4.137/
&DEVC ID='THCP54', QUANTITY='THERMOCOUPLE', XYZ=27.0,4.5,4.137/
&DEVC ID='THCP55', QUANTITY='THERMOCOUPLE', XYZ=27.5,4.5,4.137/
&DEVC ID='THCP56', QUANTITY='THERMOCOUPLE', XYZ=28.0,4.5,4.137/
&DEVC ID='THCP57', QUANTITY='THERMOCOUPLE', XYZ=28.5,4.5,4.137/
&DEVC ID='THCP58', QUANTITY='THERMOCOUPLE', XYZ=29.0,4.5,4.137/
&DEVC ID='THCP59', QUANTITY='THERMOCOUPLE', XYZ=29.5,4.5,4.137/
&DEVC ID='THCP60', QUANTITY='THERMOCOUPLE', XYZ=30.0,4.5,4.137/
&DEVC ID='THCP61', QUANTITY='THERMOCOUPLE', XYZ=30.5,4.5,4.137/
&DEVC ID='THCP62', QUANTITY='THERMOCOUPLE', XYZ=31.0,4.5,4.137/
&DEVC ID='THCP63', QUANTITY='THERMOCOUPLE', XYZ=31.5,4.5,4.137/
&DEVC ID='THCP64', QUANTITY='THERMOCOUPLE', XYZ=32.0,4.5,4.137/
&DEVC ID='THCP65', QUANTITY='THERMOCOUPLE', XYZ=32.5,4.5,4.137/
&DEVC ID='THCP66', QUANTITY='THERMOCOUPLE', XYZ=33.0,4.5,4.137/
&DEVC ID='THCP67', QUANTITY='THERMOCOUPLE', XYZ=33.5,4.5,4.137/
&DEVC ID='THCP68', QUANTITY='THERMOCOUPLE', XYZ=34.0,4.5,4.137/
&DEVC ID='THCP69', QUANTITY='THERMOCOUPLE', XYZ=34.5,4.5,4.137/
&DEVC ID='THCP70', QUANTITY='THERMOCOUPLE', XYZ=35.0,4.5,4.137/
&DEVC ID='THCP71', QUANTITY='THERMOCOUPLE', XYZ=35.5,4.5,4.137/
&DEVC ID='THCP72', QUANTITY='THERMOCOUPLE', XYZ=36.0,4.5,4.137/
&DEVC ID='THCP73', QUANTITY='THERMOCOUPLE', XYZ=36.5,4.5,4.137/
&DEVC ID='THCP74', QUANTITY='THERMOCOUPLE', XYZ=37.0,4.5,4.137/
&DEVC ID='THCP75', QUANTITY='THERMOCOUPLE', XYZ=37.5,4.5,4.137/
&DEVC ID='THCP76', QUANTITY='THERMOCOUPLE', XYZ=38.0,4.5,4.137/
&DEVC ID='THCP77', QUANTITY='THERMOCOUPLE', XYZ=38.5,4.5,4.137/
&DEVC ID='THCP78', QUANTITY='THERMOCOUPLE', XYZ=39.0,4.5,4.137/
&DEVC ID='THCP79', QUANTITY='THERMOCOUPLE', XYZ=39.5,4.5,4.137/
&DEVC ID='THCP80', QUANTITY='THERMOCOUPLE', XYZ=40.0,4.5,4.137/
&DEVC ID='THCP81', QUANTITY='THERMOCOUPLE', XYZ=40.5,4.5,4.137/
&DEVC ID='THCP82', QUANTITY='THERMOCOUPLE', XYZ=41.0,4.5,4.137/
&DEVC ID='THCP83', QUANTITY='THERMOCOUPLE', XYZ=41.5,4.5,4.137/
&DEVC ID='THCP84', QUANTITY='THERMOCOUPLE', XYZ=42.0,4.5,4.137/
&DEVC ID='THCP85', QUANTITY='THERMOCOUPLE', XYZ=42.5,4.5,4.137/
&DEVC ID='THCP86', QUANTITY='THERMOCOUPLE', XYZ=43.0,4.5,4.137/
&DEVC ID='THCP87', QUANTITY='THERMOCOUPLE', XYZ=43.5,4.5,4.137/
&DEVC ID='THCP88', QUANTITY='THERMOCOUPLE', XYZ=44.0,4.5,4.137/
&DEVC ID='THCP89', QUANTITY='THERMOCOUPLE', XYZ=44.5,4.5,4.137/
&DEVC ID='THCP90', QUANTITY='THERMOCOUPLE', XYZ=45.0,4.5,4.137/
&DEVC ID='THCP91', QUANTITY='THERMOCOUPLE', XYZ=45.5,4.5,4.137/
&DEVC ID='THCP92', QUANTITY='THERMOCOUPLE', XYZ=46.0,4.5,4.137/
&DEVC ID='THCP93', QUANTITY='THERMOCOUPLE', XYZ=46.5,4.5,4.137/
&DEVC ID='THCP94', QUANTITY='THERMOCOUPLE', XYZ=47.0,4.5,4.137/
&DEVC ID='THCP95', QUANTITY='THERMOCOUPLE', XYZ=47.5,4.5,4.137/
&DEVC ID='THCP96', QUANTITY='THERMOCOUPLE', XYZ=48.0,4.5,4.137/
&DEVC ID='THCP97', QUANTITY='THERMOCOUPLE', XYZ=48.5,4.5,4.137/
&DEVC ID='THCP98', QUANTITY='THERMOCOUPLE', XYZ=49.0,4.5,4.137/
&DEVC ID='THCP99', QUANTITY='THERMOCOUPLE', XYZ=49.5,4.5,4.137/
&DEVC ID='THCP100', QUANTITY='THERMOCOUPLE', XYZ=50.0,4.5,4.137/
&DEVC ID='THCP101', QUANTITY='THERMOCOUPLE', XYZ=50.5,4.5,4.137/
&DEVC ID='THCP102', QUANTITY='THERMOCOUPLE', XYZ=51.0,4.5,4.137/
&DEVC ID='THCP103', QUANTITY='THERMOCOUPLE', XYZ=51.5,4.5,4.137/
&DEVC ID='THCP104', QUANTITY='THERMOCOUPLE', XYZ=52.0,4.5,4.137/
&DEVC ID='THCP105', QUANTITY='THERMOCOUPLE', XYZ=52.5,4.5,4.137/
&DEVC ID='THCP106', QUANTITY='THERMOCOUPLE', XYZ=53.0,4.5,4.137/
&DEVC ID='THCP107', QUANTITY='THERMOCOUPLE', XYZ=53.5,4.5,4.137/
&DEVC ID='THCP108', QUANTITY='THERMOCOUPLE', XYZ=54.0,4.5,4.137/
&DEVC ID='THCP109', QUANTITY='THERMOCOUPLE', XYZ=54.5,4.5,4.137/
&DEVC ID='THCP110', QUANTITY='THERMOCOUPLE', XYZ=55.0,4.5,4.137/
&DEVC ID='THCP111', QUANTITY='THERMOCOUPLE', XYZ=55.5,4.5,4.137/
&DEVC ID='THCP112', QUANTITY='THERMOCOUPLE', XYZ=56.0,4.5,4.137/
&DEVC ID='THCP113', QUANTITY='THERMOCOUPLE', XYZ=56.5,4.5,4.137/
&DEVC ID='THCP114', QUANTITY='THERMOCOUPLE', XYZ=57.0,4.5,4.137/
&DEVC ID='THCP115', QUANTITY='THERMOCOUPLE', XYZ=57.5,4.5,4.137/
&DEVC ID='THCP116', QUANTITY='THERMOCOUPLE', XYZ=58.0,4.5,4.137/
&DEVC ID='THCP117', QUANTITY='THERMOCOUPLE', XYZ=58.5,4.5,4.137/
&DEVC ID='THCP118', QUANTITY='THERMOCOUPLE', XYZ=59.0,4.5,4.137/
&DEVC ID='THCP119', QUANTITY='THERMOCOUPLE', XYZ=59.5,4.5,4.137/
&DEVC ID='THCP120', QUANTITY='THERMOCOUPLE', XYZ=60.0,4.5,4.137/

&SURF ID='ignitor',
      TMP_FRONT=3000.0,
      EMISSIVITY=1.0,
      GEOMETRY='CYLINDRICAL',
      LENGTH=0.15,
      RADIUS=0.01,
      RAMP_T='IGNITION'/
&RAMP ID='IGNITION', T= 0, F=0 /
&RAMP ID='IGNITION', T= 1, F=1 /
&RAMP ID='IGNITION', T= 15, F=0 /
&SURF ID='burner',
      COLOR='RED',
      HRRPUA=10000.0,
      TMP_FRONT=300.0/
&SURF ID='concrete',
      COLOR='GRAY 60'/

&INIT ID='ignitor particle', PART_ID='ignitor particle', N_PARTICLES_PER_CELL=1, CELL_CENTERED=.TRUE., XB=9.5,10.5,4.0,5.0,0.2,0.5/ 

&OBST ID='fire', XB=9.5,10.5,4.0,5.0,0.0,0.2, SURF_IDS='burner','INERT','INERT'/ 
&OBST ID='floor', XB=0.0,60.0,0.0,9.0,0.0,0.0, COLOR='INVISIBLE', SURF_ID='concrete'/ 
&OBST ID='celling1', XB=0.0,20.0,0.0,9.0,4.2,4.2, SURF_ID='concrete'/ 
&OBST ID='celling1', XB=0.0,60.0,0.0,9.0,0.0,0.0, SURF_ID='concrete'/ 
&OBST ID='celling2', XB=21.8,60.0,0.0,9.0,4.2,4.2, SURF_ID='concrete'/ 
&OBST ID='flontwall', XB=0.0,60.0,0.0,0.0,0.0,4.2, COLOR='INVISIBLE', SURF_ID='concrete'/ 
&OBST ID='backwall', XB=0.0,60.0,9.0,9.0,0.0,4.2, SURF_ID='concrete'/ 
&OBST ID='Obstruction', XB=20.0,21.8,9.0,9.0,4.2,5.0, SURF_ID='concrete'/ 
&OBST ID='Obstruction', XB=20.0,21.8,0.0,9.0,4.698693,5.0, COLOR='INVISIBLE', SURF_ID='concrete'/ 
&OBST ID='Obstruction', XB=20.0,21.8,0.0,0.0,4.2,5.0, SURF_ID='concrete'/ 
&OBST ID='Obstruction', XB=21.8,21.8,0.0,9.0,4.2,5.0, REMOVABLE=.FALSE., SURF_ID='concrete'/ 
&OBST ID='Obstruction', XB=20.0,20.0,0.0,9.0,4.2,5.0, SURF_ID='concrete'/ 
&OBST ID='Obstruction', XB=0.0,0.0,0.0,9.0,0.0,4.2, SURF_ID='concrete'/ 

&VENT ID='Vent01', SURF_ID='OPEN', XB=60.0,60.0,0.0,9.0,0.0,4.2/ 

&BNDF QUANTITY='SOLID SPECIFIC HEAT'/
&BNDF QUANTITY='WALL TEMPERATURE'/

&SLCF QUANTITY='TEMPERATURE', PBY=4.5/
&SLCF QUANTITY='VELOCITY', VECTOR=.TRUE., PBY=4.5/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='AIR', PBY=4.5/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='OXYGEN', PBY=4.5/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='REAC_FUEL', PBY=4.5/
&SLCF QUANTITY='PRESSURE', PBY=4.5/

&DEVC ID='[Species: OXYGEN] Volume Fraction（up）_VOLUME MEAN', QUANTITY='VOLUME FRACTION', SPEC_ID='OXYGEN', SPATIAL_STATISTIC='VOLUME MEAN', XB=0.0,20.0,0.0,9.0,0.0,4.2/
&DEVC ID='[Species: OXYGEN] Volume Fraction01_VOLUME MEAN', QUANTITY='VOLUME FRACTION', SPEC_ID='OXYGEN', SPATIAL_STATISTIC='VOLUME MEAN', XB=21.8,60.0,0.0,9.0,0.0,4.2/

&TAIL /

**Comment by rmcdermo** [comment]
* SOOT_YIELD seems way too high
* REAC ID says PROPYLENE and then you give a FORMULA for heptane.
* Do not start off using CHECK_HT
* Do not use Dynamic Smagorinsky
* Modeling ignition is hard.  You are going to just be playing with the ignitor and have no real justification for your final answer except that something burned and then went out and then you will not have any good reason for claiming your suppression was physical.
* As is the case with many who start using FDS, you are starting with one of the most complicated cases possible.  You should become an expert in the fire modeling and this code before you tackle a case this hard.

**Reply by joker121-move** [reply]
> * SOOT_YIELD seems way too high
> * REAC ID says PROPYLENE and then you give a FORMULA for heptane.
> * Do not start off using CHECK_HT
> * Do not use Dynamic Smagorinsky
> * Modeling ignition is hard.  You are going to just be playing with the ignitor and have no real justification for your final answer except that something burned and then went out and then you will not have any good reason for claiming your suppression was physical.
> * As is the case with many who start using FDS, you are starting with one of the most complicated cases possible.  You should become an expert in the fire modeling and this code before you tackle a case this hard.

Thank you very much for your suggestion, I will continue to improve the model based on your valuable comments. The igniter was used because previously the flame would continue to burn at the 0°C auto-ignition temperature setting, even below 7 per cent oxygen concentration, and the igniter was able to achieve self-extinguishing.
